### PR TITLE
Remove wallet

### PR DESCRIPTION
**Pull request by sergous**
Remove wallet added to api, store (with spec), model.
Link remove added to WalletActionBar to the right (should be styled).

Known issues: on remove Uncaught (in promise) SyntaxError: Unexpected end of JSON input.

To improve: add confirm dialog before remove.

@optimusway please take a look.

### CODE REVIEW

File: src/api/restApi.ts [hunk 1]
```diff
@@ -70,6 +71,19 @@ export class RestApi {
...
+  private _delete(wretcher: () => Wretcher) {
+    return (
+      url: string,
+      cb: () => void = this.rootStore.authStore.redirectToAuthServer
+    ) =>
+      wretcher()
+        .url(url)
+        .delete()
+        .unauthorized(cb)
+        .json();
```
Comment (optimusway) [inline src/api/restApi.ts]: will it return any serializable object?

File: src/components/WalletActionBar/index.tsx [hunk 2]
```diff
@@ -32,6 +33,12 @@ export class WalletActionBar extends React.Component<WalletActionBarProps> {
             className="wallet-action-bar__item"
           />
         </ul>
+        <div
+          className="wallet-action-bar__item pull-right"
+          onClick={this.handleRemove}
+        >
+          REMOVE
```
Comment (optimusway) [inline src/components/WalletActionBar/index.tsx]: plz do uppercase with css

File: src/components/WalletActionBar/index.tsx [hunk 3]
```diff
@@ -71,6 +78,10 @@ export class WalletActionBar extends React.Component<WalletActionBarProps> {
       }, 2000);
     }
   };
+
+  private handleRemove = () => {
```
Comment (optimusway) [inline src/components/WalletActionBar/index.tsx]: another way of doing this is to define handler in the `inject` section like this https://github.com/LykkeCity/LykkeExchangeWeb/commit/46afa83bcb45d91b89cafb76084ddc22ebc47a42#diff-982702be75a4d0aff3103871c68a0141R90

plz have a look and lmk what do you think

Comment (sergous) [reply src/components/WalletActionBar/index.tsx]: I like this handler, implemented.

File: src/api/restApi.ts (hunk 1 above)
Comment (sergous) [reply src/api/restApi.ts]: no, just void. removing.

**Comment by sergous** [comment]
Updated, please take a look.

**Comment by sergous** [comment]
Updated remove api call and spec.

**Comment by sergous** [comment]
This pull request will be recreated with new one.

**Pull request closed without merging**